drivers/rtc/rtc_rv8263.c

```diff
@@ -133,6 +133,24 @@ static int rv8263c8_update_disable_timer(const struct device *dev)
 }
 
 #if (CONFIG_RTC_ALARM || CONFIG_RTC_UPDATE) && DT_ANY_INST_HAS_PROP_STATUS_OKAY(int_gpios)
+static int rv8263c8_disable_alarm(const struct device *dev)
+{
+	const struct rv8263c8_config *config = dev->config;
+
+	return i2c_reg_update_byte_dt(&config->i2c_bus, RV8263C8_REGISTER_CONTROL_2,
+				      RV8263C8_BM_ALARM_INT_ENABLE, RV8263C8_BM_ALARM_INT_DISABLE) |
+	       i2c_reg_write_byte_dt(&config->i2c_bus, RV8263C8_REGISTER_SECONDS_ALARM,
+				     RV8263C8_BM_ALARM_DISABLE) |
+	       i2c_reg_write_byte_dt(&config->i2c_bus, RV8263C8_REGISTER_MINUTES_ALARM,
+				     RV8263C8_BM_ALARM_DISABLE) |
+	       i2c_reg_write_byte_dt(&config->i2c_bus, RV8263C8_REGISTER_HOURS_ALARM,
+				     RV8263C8_BM_ALARM_DISABLE) |
+	       i2c_reg_write_byte_dt(&config->i2c_bus, RV8263C8_REGISTER_DATE_ALARM,
+				     RV8263C8_BM_ALARM_DISABLE) |
+	       i2c_reg_write_byte_dt(&config->i2c_bus, RV8263C8_REGISTER_WEEKDAY_ALARM,
+				     RV8263C8_BM_ALARM_DISABLE);
```

Comment:
weird... do the calls one at a time and check for each one, no sense cutting this short and letting a failed transaction fall back to the next one, also the indentation looks odd